backend/lcfs/web/api/compliance_report/summary_service.py

```diff
@@ -720,11 +734,10 @@ async def calculate_fuel_quantities(
             await self.repo.aggregate_other_uses(compliance_report_id, fossil_derived)
         )
 
-        if not fossil_derived:
-            fuel_quantities.update(
-                await self.repo.aggregate_allocation_agreements(compliance_report_id)
-            )
-
+        # if not fossil_derived:
```

Comment:
just remove this to avoid confusion if this was the only material change to the calculation.